docs/source/llms/transformers/task.rst

```diff
@@ -0,0 +1,208 @@
+     - ``text-generation``
+     - `Completions API spec <https://mlflow.org/docs/latest/llms/deployments/index.html#completions>`_
+     - Returns a `Completion <https://platform.openai.com/docs/guides/text-generation/completions-api>`_ object in the json format.
+   * - ``llm/v1/embeddings``
+     - ``feature-extraction``
+     - `Embeddings API spec <https://mlflow.org/docs/latest/llms/deployments/index.html#embeddings>`_
+     - Returns a list of `Embedding <https://platform.openai.com/docs/api-reference/embeddings/object>`_ object. Additionally, the model returns ``usage`` field, which contains the number of tokens used for the embeddings generation.
+
+.. note::
+
+    The Completion API is considered as legacy, but it is still supported in MLflow for backward compatibility. We recommend using the Chat API for compatible with the latest APIs from OpenAI and other model providers.
+
+Code Example of Using ``llm/v1`` Tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following code snippet demonstrates how to log a Transformers pipeline with the ``llm/v1/chat`` task type, and use the model for chat-style inference. Check out the `notebook tutorial <../tutorials/conversational/pyfunc-chat-model.html>`_ to see more examples in action!
+
+.. code-block:: python
+
+    import mlflow
+    import transformers
+
+    pipeline = transformers.pipeline("text-generation", "gpt2")
+
+    with mlflow.start_run():
+        model_info = mlflow.transformers.log_model(
+            transformers_model=pipeline,
+            artifact_path="model",
+            task="llm/v1/chat",
+            input_example={
+                "messages": [
+                    {"role": "system", "content": "You are a bot."},
+                    {"role": "user", "content": "Hello, how are you?"},
+                ]
+            },
+            save_pretrained=False,
+        )
+
+    # Model metadata logs additional field "inference_task"
+    print(model_info.flavors["transformers"]["inference_task"])
+    # >> llm/v1/chat
+
+    # The original native task type is also saved
+    print(model_info.flavors["transformers"]["task"])
+    # >> text-generation
+
+    # Model signature is set to the chat API spec
+    print(model_info.signature)
+    # >> inputs:
+    # >>   ['messages': Array({content: string (required), name: string (optional), role: string (required)}) (required), 'temperature': double (optional), 'max_tokens': long (optional), 'stop': Array(string) (optional), 'n': long (optional), 'stream': boolean (optional)]
+    # >> outputs:
+    # >>   ['id': string (required), 'object': string (required), 'created': long (required), 'model': string (required), 'choices': Array({finish_reason: string (required), index: long (required), message: {content: string (required), name: string (optional), role: string (required)} (required)}) (required), 'usage': {completion_tokens: long (required), prompt_tokens: long (required), total_tokens: long (required)} (required)]
```

Comment:
/ai format this text

Reply:
not as readable as I expected 😅

Reply:
yea.. I tried to format it, but can't find a good way that is readable while not taking too long horizontal space.,,